README.md

```diff
@@ -41,6 +41,25 @@ pip install tox
 tox
 ```
 
+### Docker Image
+You can also directly use TAPAS via Docker.
+
+For first time use of nvidia-docker, it is important to run the following commands:
+
+```
+distribution=$(. /etc/os-release;echo $ID$VERSION_ID)
+curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey | sudo apt-key add -
+curl -s -L https://nvidia.github.io/nvidia-docker/$distribution/nvidia-docker.list | sudo tee /etc/apt/sources.list.d/nvidia-docker.list
+sudo apt-get update && sudo apt-get install -y nvidia-container-toolkit
+sudo systemctl restart docker
+```
+
+To pull and run the docker image (contains all prerequisites) you can call the following command:
+
+```  
+sudo docker run -it --gpus all -v /absolute/path/to/my/data/directory:/home/data oeslle/tapas:1.0
```

Comment:
I think we would prefer to include the Dockerfile directly in the repository instad of linking to an external source

Reply:
Hi @eisenjulian I added the Dockerfile. However, due to issue #33 `tensorflow-probability==0.10.1` must be in the `requirements.txt` file and `tox.ini` should have `envlist = py36` only.